content/en/docs/concepts/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Concepts
+description: The collection of requirements and scenarios that define v2 of the Notary project
+weight: 6
```

Comment:
Can we adjust the weight of the Concepts chapter to 2 or 3? Take [Istio](https://istio.io/latest/docs/) and [Kubernetes](https://kubernetes.io/docs/home/) as examples. Now I think the conceptual part is that people might need to understand in advance.